.github/CODEOWNERS

```diff
@@ -1 +1,2 @@
-/content/accessibility/ @primer/accessibility-reviewers
\ No newline at end of file
+/content/accessibility/ @primer/accessibility-reviewers
+*                       @primer/reviewers
```

Comment:
reviewers is everyone on the team, do we need that many?

Reply:
what do we do for other repos?

Reply:
would be nice to narrow it down, should we create a "design-reviewers" team?

Reply:
> what do we do for other repos?

We have `css-reviwers`, `react-reviewers`, etc: https://github.com/orgs/primer/teams